Server_Side_TLS.mediawiki

```diff
@@ -1,600 +1,236 @@
+
+{| class="wikitable" style="margin: 1.5rem 1rem;"
+|-
+! Configuration
+! Firefox
+! Android
+! Chrome
+! Edge
+! Internet Explorer
+! Java
+! OpenSSL
+! Opera
+! Safari
+|-
+| style="color: green;" | '''Modern'''
+| style="text-align: center;" | 63
+| style="text-align: center;" | 10.0
+| style="text-align: center;" | 70
+| style="text-align: center;" | 75
+| style="text-align: center;" | --
+| style="text-align: center;" | 11
+| style="text-align: center;" | 1.1.1
+| style="text-align: center;" | 57
+| style="text-align: center;" | 12.1
+|-
+| style="color:orange;" | '''Intermediate'''
+| style="text-align: center;" | 27
+| style="text-align: center;" | 4.4.2
+| style="text-align: center;" | 31
+| style="text-align: center;" | 12
+| style="text-align: center;" | 11 (Win7)
+| style="text-align: center;" | 8u31
+| style="text-align: center;" | 1.0.1
+| style="text-align: center;" | 20
+| style="text-align: center;" | 9
+|-
+| style="color:gray;" | '''Old'''
+| style="text-align: center;" | 1
+| style="text-align: center;" | 2.3
+| style="text-align: center;" | 1
+| style="text-align: center;" | 12
+| style="text-align: center;" | 8 (WinXP)
+| style="text-align: center;" | 6
+| style="text-align: center;" | 0.9.8
+| style="text-align: center;" | 5
+| style="text-align: center;" | 1
 |}
 
+<p style="max-width: 60em;">The ordering of cipher suites in the <span style="color: orange; font-weight: bold;">Intermediate</span> and <span style="color: gray; font-weight: bold;">Old</span> configurations is very important, as it determines the priority with which algorithms are selected.</p>
 
-Older versions of OpenSSL may not return the full list of algorithms. AES-GCM and some ECDHE are fairly recent, and not present on most versions of OpenSSL shipped with Ubuntu or RHEL. This listing below was obtained from a freshly built OpenSSL. If your version of OpenSSL is old, unavailable ciphers will be discarded automatically. Always use the full ciphersuite and let OpenSSL pick the ones it supports.
-
-The ordering of a ciphersuite is very important because it decides which algorithms are going to be selected in priority. Each level shows the list of algorithms returned by its ciphersuite. If you have to pick ciphers manually for your application, make sure you keep the ordering.
-
-The ciphersuite numbers listed come from the IANA [https://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml#tls-parameters-4 TLS Cipher Suite Registry]. Previous versions of these recommendations included draft numbers for ECDHE-ECDSA-CHACHA20-POLY1305 (0xCC,0x14) and ECDHE-RSA-CHACHA20-POLY1305 (0xCC,0x13).
+<p style="max-width: 60em;">OpenSSL will ignore cipher suites it doesn't understand, so always use the full set of cipher suites below, in their recommended order. The use of the <span style="color: gray; font-weight: bold;">Old</span> configuration with modern versions of OpenSSL may require custom builds with support for SSLv3 and deprecated ciphers.</p>
```

Comment:
Why do you mention enabling SSLv3 via custom builds, when SSLv2 and SSLv3 will be abandoned by this guide after this PR is merged? However a custom OpenSSL build for TLSv1.0 and TLSv1.1 might be necessary (e.g. for upcoming Debian 10).

Reply:
Because you have to enable SSLv3 in the build to get access to 3DES.  :)

Reply:
That said, I can see how that would be confusing, so I'll trim it up a little bit.